### ISSUE TITLE
issue with publish-for-arch, externalify

### ISSUE THREAD

**Issue by markoshust** [meteor/meteor#5636]
I'm having an issue with building an NPM wrapper package that apparently contains binary code. I executed the proper publish-for-arch commands, however I'm seeing an externalify error when trying to install the package on a fresh meteor project:

```
[[[[[ ~/web-dev/Meteor/adrian-test/mealtube ]]]]]

=> Started proxy.                             
=> Started MongoDB.                           

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Cannot find module 'externalify' from '/Users/hl/web-dev/Meteor/adrian-test/mealtube'
  at /Users/hl/.meteor/packages/cosmos_browserify/.0.8.3.1jl1ity++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:46:17
  at process (/Users/hl/.meteor/packages/cosmos_browserify/.0.8.3.1jl1ity++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:173:43)
  at ondir (/Users/hl/.meteor/packages/cosmos_browserify/.0.8.3.1jl1ity++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:188:17)
  at load (/Users/hl/.meteor/packages/cosmos_browserify/.0.8.3.1jl1ity++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
  at onex (/Users/hl/.meteor/packages/cosmos_browserify/.0.8.3.1jl1ity++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
  at /Users/hl/.meteor/packages/cosmos_browserify/.0.8.3.1jl1ity++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
  at Object.oncomplete (fs.js:108:15)
```

This error happens on OS X, the same platform I've built the package on. If I check out the same code from GitHub, and add it to a local package at `packages/radium`, I receive no error.

The full back history on this is at https://github.com/markoshust/meteor-radium/issues/5#issuecomment-154430488

**Comment by stubailo** [comment]
That's very interesting, it's hard to see why that is happening. Perhaps, file the issue on cosmos:browserify and see if @elidoran knows what this means?

**Comment by elidoran** [comment]
I'll add a reply to the referenced issue.

In short cosmos:browserify appears to be looking in the root of the app instead of where published packages are stored, *~/.meteor/packages...*, assuming the package being added is published and not local. It's not looking where the local package is kept either.

@stubailo I would be much happier if I could get the location of published packages in a far easier way than [this](https://github.com/elidoran/cosmos-browserify/blob/8ec5ad46e5156cbea369a9224971e0f9c7ba1da2/plugin/browserify.coffee#L45). Many deep properties just to find the stuff.

**Comment by elidoran** [comment]
Fixed in 0.8.4+

**Comment by markoshust** [comment]
Thanks, this was a prob with cosmos:browserify. All fixed with 0.8.4.
